backend/src/main/java/com/javahelp/backend/endpoint/HTTPTokenHandler.java

```diff
@@ -99,8 +100,8 @@ private boolean extractTokenAndUserID(String[] authHeader) {
         for (String s : authHeader) {
             if (s.startsWith("id=")) {
                 userId = s.substring(3);
-            } else if (s.startsWith("username=")) {
-                tokenString = s.substring(9);
+            } else if (s.startsWith("token=")) {
+                tokenString = s.substring(6);
```

Comment:
When we passed the information into the handler it comes in the form of a HTTP header (as you can see in the tests), where the ID for the token is specified after `token=` in the header (there are 6 letters/= in `token=` so the actual ID starts at index 6). It was 9 previously because Jacob accidentally typed `username=` (where there are 9 letters/= in it) so I was fixing that typo.